src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+
+    @Override
+    public OrderResponseDto toDto(Order object) {
+        OrderResponseDto orderResponseDto = new OrderResponseDto();
+        orderResponseDto.setId(object.getId());
+        orderResponseDto.setTicketsId(object.getTickets()
+                .stream()
+                .map(t -> t.getId())

```

Comment:
Gues that you can replace here and in other mapper lambda to method reference)

Reply:
and let's do this also